readthedocs_build/config/test_config.py

```diff
@@ -281,23 +281,6 @@ def it_validates_wrong_type_right_value():
         build.validate_python()
         assert build['python']['version'] == 3
 
-    def it_validates_env_supported_versions():
-        build = get_build_config(
-            {'python': {'version': 3.6}},
-            env_config={'python': {'supported_versions': [3.5]}}
-        )
-        with raises(InvalidConfig) as excinfo:
-            build.validate_python()
-        assert excinfo.value.key == 'python.version'
-        assert excinfo.value.code == INVALID_CHOICE
-
-        build = get_build_config(
-            {'python': {'version': 3.6}},
-            env_config={'python': {'supported_versions': [3.5, 3.6]}}
-        )
-        build.validate_python()
-        assert build['python']['version'] == 3.6
```

Comment:
Seems we should make this error work instead of removing it.  We use supported versions for giving intelligent errors to users.